assets/app/view/game/round/auction.rb

```diff
@@ -13,6 +13,7 @@ class Auction < Snabberb::Component
         include Actionable
 
         needs :selected_company, default: nil, store: true
+        needs :selected_corporation, default: nil, store: true
```

Comment:
i don't think you need this

Reply:
I need "selected_corporation", otherwise I can't respond to a Corporation UI element being clicked.

Reply:
when do you respond to a corporation being clicked though? isn't it auto populated through step.auctioning?

Reply:
It's populated, but I don't show the "Par" buttons until the corp has been clicked. No different that the private companies not showing the "Buy" button until the company has been clicked.

Reply:
but you don't need to actually click it, you know which one is gonna be, so no need for another user action unless i'm misunderstanding

Reply:
You have to click a company or corporation first in order to bid/buy/par/etc it. Otherwise you don't get the "input" controls. At the beginning of the round, there are six choices. It isn't until it is clicked that the view "knows" which company/corp you are buying/parring and attaches the input.

Reply:
And in fact, during the turn auction, I don't want the companies/corps to be clicked, since they aren't being bid on.